certbot-dns-linode/certbot_dns_linode/__init__.py

```diff
@@ -74,13 +74,13 @@
      -d www.example.com
 
 .. code-block:: bash
-   :caption: To acquire a certificate for ``example.com``, waiting 60 seconds
+   :caption: To acquire a certificate for ``example.com``, waiting 1000 seconds
              for DNS propagation
 
    certbot certonly \\
      --dns-linode \\
      --dns-linode-credentials ~/.secrets/certbot/linode.ini \\
-     --dns-linode-propagation-seconds 60 \\
+     --dns-linode-propagation-seconds 1000 \\
```

Comment:
I'm okay with the change in the example to reflect how long people should wait, but maybe we should also motivate why (cite linode's quarter-hour zone republishing)

Reply:
Commit amended to add comments explaining both the default value and the example value.